src/utils/test.ts

```diff
@@ -58,3 +65,67 @@ export function withTestScheduler(
     }),
   );
 }
+
+export async function withLocalMedia(
```

Comment:
Would it make sense to not require tests (ignore test utils in codecov) on our test utils?